src/stackset-stack.ts

```diff
@@ -49,9 +75,10 @@ export class StackSetStack extends Stack {
   public readonly templateFile: string;
   private _templateUrl?: string;
   private _parentStack: Stack;
-  constructor(scope: Construct, id: string) {
+  constructor(scope: Construct, id: string, props: StackSetStackProps) {
```

Comment:
```suggestion
  constructor(scope: Construct, id: string, props: StackSetStackProps = {}) {
```

Sorry 1 more. Since all the props are optional this should be optional as well.